docs/get-started/frameworks/nextjs.mdx

```diff
@@ -78,7 +78,7 @@ Finally, if you were using Storybook plugins to integrate with Next.js, those ar
 
 (⚠️ **Experimental**)
```

Comment:
Should this line be removed?

Reply:
I haven't touched the docs changes at all. I leave this to @jonniebigodes